README.md

```diff
@@ -19,6 +19,20 @@ that incorporates and documents best practices for a performant and scalable des
 security by default for control, logging and evidence generation. It can be  simply deployed by
 customers through a Terraform workflow.
 
+## Security Recomendations
+
+We recommend you remove all primitive `owner` roles in the projects used as inputs for the *Data Warehouse module*.
```

Comment:
Consider adding the following
"When using this blueprint, it’s important to understand how you manage separation of duties.   We recommend you remove....  The secured data warehouse itself does not need any primitive `owner` roles for correct operations.”

When using this blueprint in the example mode or when using this blueprint to create the new projects with default configurations for the deployment, we automatically remove the `owner` role as it has too broad access.

However, if you choose to use this blueprint with pre-existing projects in your organization, we will not proactively remove any pre-existing `owner` role assignments, as we won’t know your intent for or dependency on these role assignments in your pre-existing workloads.  The pre-existing presence of these roles does expand the attack and risk surface of the resulting deployment.  Therefore, we highly recommend you review your use of `owner` roles in these pre-existing cases and see if you can eliminate them to improve your resulting security posture.  Only you can determine the appropriate trade-off to meet your business requirements.

Reply:
Good points, I will add to the text.